<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInstallerActivity</class>
 <widget class="QWidget" name="GameInstallerActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>594</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mp_layout">
     <item>
      <widget class="QPushButton" name="mp_btn_back">
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset theme="go-previous" resource="../Resources/Resources.qrc">
         <normaloff>:/go-previous</normaloff>:/go-previous</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="mp_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_install">
       <property name="text">
        <string>&amp;Install</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start" resource="../Resources/Resources.qrc">
         <normaloff>:/media-playback-start</normaloff>:/media-playback-start</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="mp_progressbar_overall">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mp_layout_4">
     <item>
      <widget class="QWidget" name="mp_widget_tasks" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>14</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="mp_tree_tasks">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>7</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>300</number>
          </attribute>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mp_layout_2">
          <item>
           <widget class="QPushButton" name="mp_btn_add_image">
            <property name="text">
             <string>&amp;Add Image</string>
            </property>
            <property name="icon">
             <iconset theme="list-add" resource="../Resources/Resources.qrc">
              <normaloff>:/list-add</normaloff>:/list-add</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_btn_add_disc">
            <property name="text">
             <string>Add &amp;Disc</string>
            </property>
            <property name="icon">
             <iconset theme="media-optical" resource="../Resources/Resources.qrc">
              <normaloff>:/media-optical</normaloff>:/media-optical</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="mp_spacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="mp_btn_remove">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset theme="edit-delete" resource="../Resources/Resources.qrc">
              <normaloff>:/edit-delete</normaloff>:/edit-delete</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_btn_rename">
            <property name="text">
             <string>Rename</string>
            </property>
            <property name="icon">
             <iconset theme="document-edit" resource="../Resources/Resources.qrc">
              <normaloff>:/document-edit</normaloff>:/document-edit</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="mp_widget_task_details_holder" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="mp_widget_task_details" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="mp_layout_3">
             <item>
              <widget class="QLabel" name="mp_label_title">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string notr="true">The title</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="mp_groupbox_media_type">
               <property name="title">
                <string>Media Type</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QRadioButton" name="mp_radio_mtauto">
                  <property name="text">
                   <string>Auto</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="mp_radio_mtdvd">
                  <property name="text">
                   <string>DVD</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="mp_radio_mtcd">
                  <property name="text">
                   <string>CD</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="mp_groupbox_options">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QRadioButton" name="mp_radio_split_up">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It is the only way to run large games (larger than 4 GiB) from a USB stick. Since the OPL supports only the FAT32 file system, you should split up large files into parts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="text">
                   <string>Split up ISO file into parts</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="mp_radio_dnot_split_up">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It is the recommended method if you planning to run games through a local network. The ISO files will be placed in the DVD or the CD directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="text">
                   <string>Do not split up ISO file</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="mp_checkbox_move">
                  <property name="text">
                   <string>Move ISO file instead of copy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="mp_checkbox_rename">
                  <property name="text">
                   <string>Add a game ID to filename</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mp_label_error_message">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel {
	color: rgb(255, 87, 90);
}</string>
               </property>
               <property name="text">
                <string notr="true">Error message</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="mp_spacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_btn_back</tabstop>
  <tabstop>mp_btn_install</tabstop>
  <tabstop>mp_btn_cancel</tabstop>
  <tabstop>mp_tree_tasks</tabstop>
  <tabstop>mp_radio_mtauto</tabstop>
  <tabstop>mp_radio_mtdvd</tabstop>
  <tabstop>mp_radio_mtcd</tabstop>
  <tabstop>mp_radio_split_up</tabstop>
  <tabstop>mp_radio_dnot_split_up</tabstop>
  <tabstop>mp_checkbox_move</tabstop>
  <tabstop>mp_checkbox_rename</tabstop>
  <tabstop>mp_btn_add_image</tabstop>
  <tabstop>mp_btn_add_disc</tabstop>
  <tabstop>mp_btn_remove</tabstop>
  <tabstop>mp_btn_rename</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
